pyarlo/__init__.py

```diff
@@ -99,6 +99,8 @@ def devices(self):
             name = device.get('deviceName')
             if device.get('deviceType') == 'camera':
                 devices['cameras'].append(ArloCamera(name, device, self))
+            if device.get('deviceType') == 'arloqs':
```

Comment:
Hello @aneisch, thank you for your contribution. 
Could you please update your PR to test if the device is an Arlo Q on the same line to avoid the duplicate `devices['cameras'].append` line? 
```python
if device.get('deviceType') == 'camera' or device.get('deviceType') == 'arloqs':
    devices['cameras'].append(ArloCamera(name, device, self))
```